lib/js/components/Modal/Modal.vue

```diff
@@ -66,14 +66,18 @@
 
 	&__header {
 		align-items: center;
-		box-shadow: 0 $space-xxxxs $space-xxxxs $color-total-white;
+		box-shadow: 0 $space-xxxxs $space-xxxxs var(--raw-white);
```

Comment:
did you confirm with UX that we can use raw color here?

Reply:
No. Modals have still `#FFFFFF` color: https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS---Components?node-id=899%3A23641
So usign raw white here I think it's ok. Probably modals will have tokens in the future, but now I don't see them in the figma, so raw is the best option.